sdk/tables/azure-data-tables/inc/azure/data/tables/cryptography/hmacsha256.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "azure/core/azure_assert.hpp"
+
+#include <memory>
+#include <stdexcept>
+#include <stdint.h>
+#include <string>
+#include <vector>
+
+namespace Azure { namespace Data { namespace Tables { namespace _detail { namespace Cryptography {

```

Comment:
This file does only have definitions in `_detail` namespace, so move it to `inc/.../internal/`